Using Empty Dictionary for log_to_wandb in AttackArgs throws assertion Error.

**Describe the bug**
In AttackArgs , if we specify empty dictionary for log_to_wandb as mentioned in the [Doc](https://textattack.readthedocs.io/en/latest/api/attacker.html#attackargs) , it throws AssertionError 

**To Reproduce**
Run Code 
```
# Create the recipe

attacker = Attacker(textfooler_recipe, sst2_dataset,AttackArgs(log_to_wandb = {}))

results = attacker.attack_dataset()
```

**Expected behavior**
Empty Dictionary to log_to_wandb should work as mentioned in documentation , wandb should create project name by default.

**Screenshots or Traceback**
```
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
Input In [28], in <cell line: 5>()
      1 # Create the recipe
      3 attacker = Attacker(textfooler_recipe, sst2_dataset,AttackArgs(log_to_wandb = {}))
----> 5 results = attacker.attack_dataset()

File ~/ADL_new/lib/python3.8/site-packages/textattack/attacker.py:418, in Attacker.attack_dataset(self)
    415     self.attack.goal_function.query_budget = self.attack_args.query_budget
    417 if not self.attack_log_manager:
--> 418     self.attack_log_manager = AttackArgs.create_loggers_from_args(
    419         self.attack_args
    420     )
    422 textattack.shared.utils.set_seed(self.attack_args.random_seed)
    423 if self.dataset.shuffled and self.attack_args.checkpoint_interval:
    424     # Not allowed b/c we cannot recover order of shuffled data

File ~/ADL_new/lib/python3.8/site-packages/textattack/attack_args.py:427, in AttackArgs.create_loggers_from_args(cls, args)
    425 # Weights & Biases
    426 if args.log_to_wandb is not None:
--> 427     attack_log_manager.enable_wandb(**args.log_to_wandb)
    429 # Stdout
    430 if not args.disable_stdout and not sys.stdout.isatty():

File ~/ADL_new/lib/python3.8/site-packages/textattack/loggers/attack_log_manager.py:31, in AttackLogManager.enable_wandb(self, **kwargs)
     30 def enable_wandb(self, **kwargs):
---> 31     self.loggers.append(WeightsAndBiasesLogger(**kwargs))

File ~/ADL_new/lib/python3.8/site-packages/textattack/loggers/weights_and_biases_logger.py:16, in WeightsAndBiasesLogger.__init__(self, **kwargs)
     12 class WeightsAndBiasesLogger(Logger):
     13     """Logs attack results to Weights & Biases."""
     15     def __init__(self, **kwargs):
---> 16 #         assert "project" in kwargs
     18         global wandb
     19         wandb = LazyLoader("wandb", globals(), "wandb")

AssertionError: 
```


**System Information (please complete the following information):**
 - OS : Ubuntu (Ubuntu 18.04.6 LTS )
 - Library versions 
    transformers==4.12.2
    wandb==0.12.16
 - Textattack version
    textattack==0.3.5

**Additional context**

Fix Should just be to remove the following lines from `class WeightsAndBiasesLogger` in `textattack/loggers/weights_and_biases_logger.py` 

```
   def __init__(self, **kwargs):
        assert "project" in kwargs               # Remove this line 

        global wandb
        wandb = LazyLoader("wandb", globals(), "wandb")

        wandb.init(**kwargs)
        self.kwargs = kwargs
        self.project_name = kwargs["project"]   # Replace  kwargs["project"] with  wandb.run.project_name()
        self._result_table_rows = []
```

--- a/textattack/loggers/weights_and_biases_logger.py
+++ b/textattack/loggers/weights_and_biases_logger.py
@@ -13,14 +13,13 @@ class WeightsAndBiasesLogger(Logger):
     """Logs attack results to Weights & Biases."""
 
     def __init__(self, **kwargs):
-        assert "project" in kwargs
 
         global wandb
         wandb = LazyLoader("wandb", globals(), "wandb")
 
         wandb.init(**kwargs)
         self.kwargs = kwargs
-        self.project_name = kwargs["project"]
+        self.project_name = wandb.run.project_name()
         self._result_table_rows = []
 
     def __setstate__(self, state):

--- a/textattack/loggers/weights_and_biases_logger.py
+++ b/textattack/loggers/weights_and_biases_logger.py
@@ -32,6 +32,14 @@ class WeightsAndBiasesLogger(Logger):
     def log_summary_rows(self, rows, title, window_id):
         table = wandb.Table(columns=["Attack Results", ""])
         for row in rows:
+            if isinstance(row[1], str):
+                try:
+                    row[1] = row[1].replace("%", "")
+                    row[1] = float(row[1])
+                except ValueError:
+                    raise ValueError(
+                        f'Unable to convert row value "{row[1]}" for Attack Result "{row[0]}" into float'
+                    )
             table.add_data(*row)
             metric_name, metric_score = row
             wandb.run.summary[metric_name] = metric_score

